troubadix/plugins/dependencies.py

```diff
@@ -58,9 +58,13 @@ def run(
             if match:
                 # Remove single and/or double quotes, spaces
                 # and create a list by using the comma as a separator
-                dependencies = re.sub(
-                    r'[\'"\s]', "", match.group("value")
-                ).split(",")
+                # additionally, check and filter for inline comments
+                dependencies = []
+
+                for line in match.group("value").splitlines():
+                    subject = line[: line.find("#")] if "#" in line else line
```

Comment:
```suggestion
                    subject = line[: line.index("#")] if "#" in line else line
```
nitpick: Can we use `index` instead? I think it clarifies a bit more, what it done here and we're checking for the existence of `#` anyways.

Reply:
Yes